app/services/CheckoutService.scala

```diff
@@ -2,25 +2,33 @@ package services
 
 import com.gu.identity.play.IdMinimalUser
 import com.gu.membership.salesforce.MemberId
+import com.gu.membership.zuora.soap.Zuora.SubscribeResult
 import com.typesafe.scalalogging.LazyLogging
 import model.SubscriptionData
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
+case class Subscription(memberId: MemberId, zuoraSubscription: SubscribeResult)
```

Comment:
It should be fairly easy to reconcile. We should just agree on the name of the case class :)